src/civis/utils/_jobs.py

```diff
@@ -1,10 +1,18 @@
 import logging
+import operator
+import time
+from datetime import datetime
 
 from civis import APIClient
 from civis.futures import CivisFuture
 
 log = logging.getLogger(__name__)
 
+_FOLLOW_POLL_INTERVAL_SEC = 5
```

Comment:
I increased the interval from 3 to 5 seconds here since there's an extra API call to check whether the job is done. Also, 3 seemed faster than necessary anyway.